test/spectral/tests/enabled-rules.test.js

```diff
@@ -402,4 +450,48 @@ describe('spectral - test enabled rules - OAS3 In Memory', function() {
       '`number_of_coins` property type should be integer'
     );
   });
+
+  it('test major-version-in-path flags multiple versions in server URLs', async () => {
```

Comment:
This test as well may be better off in `test/spectral/tests/custom-rules`.

Reply:
Fixed.